insights/parsers/tests/test_systemctl_status_opflex.py

```diff
@@ -0,0 +1,28 @@
+# coding: utf-8
+
+from insights.parsers.systemctl_status import SystemctlStatusOpflex
+from insights.tests import context_wrap
+
+SYSTEMCTL_STATUS_OPFLEX_ACTIVE = """
+neutron-opflex-agent - Open vSwitch Internal Unit
+   Loaded: loaded (/usr/lib/systemd/system/neutron-opflex-agent; static; vendor preset: disabled)
+   Active: active (exited) since Fri 2017-04-14 00:27:45 UTC; 7s ago
+   CGroup: /system.slice/neutron-opflex-agent
+""".strip()
+
+SYSTEMCTL_STATUS_OPFLEX_INACTIVE = """
+neutron-opflex-agent - Open vSwitch Internal Unit
+   Loaded: loaded (/usr/lib/systemd/system/neutron-opflex-agent; static; vendor preset: disabled)
+   Active: inactive (dead) since Fri 2017-04-14 00:23:48 UTC; 3min 17s ago
+   Main PID: 940 (code=exited, status=0/SUCCESS)
+""".strip()
+
+
+def test_systemctl_status_opflex():
+    context = context_wrap(SYSTEMCTL_STATUS_OPFLEX_ACTIVE)
+    result = SystemctlStatusOpflex(context).data
```

Comment:
Since "SystemctlStatus" is also based on "LegacyItemAccess", we can access its content directly without via ".data"